### PR TITLE
Update user map for Jul-Sep 2016 #6107

### LINKED ISSUE #6107: Update user map for Jul-Sep 2016
Here are the entries to consider:

```
Armenia American University of Armenia
Australia   Avondale College of Higher Education
Australia   Canterbury Primary
Australia   Edith Cowan University
Australia   Mandurah Catholic Colege
Australia   Monash University
Australia   Swinburne University
Australia   University of Melbourne
Australia   University of Melbourne
Australia   University of Technology, Sydney
Australia   University of Western Australia
Australia   Waverley Christian College
Australia   Western Sydney University
Belgium IMI - Roosdaal
Belgium Sint-Willebrord Heilige Familie
Belgium Solvay Brussels School of Economics & Management
Belgium Solvay Brussels School of Economics & Management
Brazil  IFSULDEMINAS
Brazil  UFMG
Canada  Algonquin College
Canada  Dalhousie University
Canada  Memorial University of Newfoundland
Canada  Ryerson University
Canada  Seneca College
Canada  Sheridan College
Canada  University of Toronto
Canada  University of Waterloo
Canada  UQAC-NAD
Canada  Wilfrid Laurier University
Canada  Sheridan College
Colombia    Universidad de Cundinamarca 
Czech Republic  Charles University in Prague
Finland Nurmon yläaste
France  Sciences Po
France  SKEMA Business School
Germany Refugee Innovation Challenge
Germany Rheinische Fachhochschule Köln
Germany Technical University of Munich 
Germany Uni Hamburg
Ghana   Ashesi University College
Guatemala   Francisco Marroquín University
Guatemala   Francisco Marroquín University
HK  HKU
Hong Kong   The ISF Academy
Hong Kong   University of Hong Kong
Hong Kong   YOT Chan Wong Shuk Fong Memorial Secondary School
Hungary UNHCR Global Learning Center
Iceland Hagaskóli
India   Ahmedabad University
India   American School Of Bombay
India   Doon Business School
India   Mumbai University
India   Rajalakshmi School of Business
Indonesia   AIESEC Bandung
Ireland Royal College of Surgeons in Ireland
Ireland University College Dublin
Italy   IIS Bruno-Franchetti
Jamaica University of The West Indies, Mona
Latvia  BA School of Banking
Malaysia    Taylor's University
Malaysia    Universiti Teknologi Malaysia
Malaysia    Universiti Teknologi PETRONAS
Netherlands Amsterdam University of Applied Science
Netherlands Amsterdam University of Applied Sciences
Netherlands Delft University of Technology
Netherlands Mhc Bommelerwaard
Netherlands ROC van Twente
Netherlands The Hague School of Applied Science
Netherlands University of Groningen
Netherlands VU University
New Zealand Auckland University of Technology
New Zealand Manukau Institute of Technology
New Zealand University of Canterbury
Peru    Independencia
Peru    Universidad Nacional de San Agustin
Peru    UNIVERSIDAD NACIONAL DE SAN AGUSTIN
Philippines Ateneo School of Medicine and Public Health
Philippines De La Salle Health Sciences Institute
Philippines Misamis Oriental Institute Of Science and Technology
Romania Senior Interactive Academy
Rwanda  University of Rwanda
Saudi Arabia    American International School of Jeddah
Saudi Arabia    Prince Sattam bin Abdulaziz University 
Singapore   Dunman High School
Singapore   Fellowship of Christian Athletes Singapore
Singapore   ITE
Singapore   Ministry of Education
Singapore   Nanyang Polytechnic
Singapore   National Institute of Education
Singapore   Rosyth School
Singapore   Serangoon Junior College
Singapore   Singapore Institute of Technology
Singapore   Singapore Polytechnic
Singapore   Singapore Polytechnic
Singapore   Singapore Polytechnic
Singapore   St Gabriels' Secondary
Singapore   ST MARGARET'S SECONDARY SCHOOL
Singapore   Temasek Junior College
Singapore   Temasek Polytechnic
Singapore   West Spring Primary School
Singapore   Woodgrove Primary School
Singapore   Singapore Polytechnic
South Africa    Pearson Institute of Higher Education
South Korea Korea Unuversity
Spain   Universidad de Valencia
Sri Lanka   University of Kelaniya
Sweden  Umeå University
UK  Coventry University
UK  Henley Business School
UK  Kingston University
UK  Manchester Metropolitan University
UK  Quarrydale Academy
UK  University of Bath
UK  University of Glasgow
UK  University of Salford
UK  University of the West of England
UAE Hult International Business School
USA Ashland University
USA Baylor University
USA Baylor University
USA Baylor University 
USA Bloomfield College
USA Bradley University
USA Brazosport College
USA Brigham Young University-Idaho
USA Broad Institute
USA Brooklyn High School
USA Bryant University
USA Bryant University
USA California State University Long Beach
USA Carroll University 
USA Cauldron Ice Cream, Inc.
USA Charles Darwin Elementary School
USA Chester County High School
USA Chestnut Hill College
USA Clark University
USA Clayton Valley Charter High School
USA Coastal Carolina University
USA College of Mount Saint Vincent
USA Columbia Middle School
USA Columbus State Community College
USA Crooms Academy of Information Technology
USA D'Youville College School of Pharmacy
USA DePaul University
USA Donald H Eichhorn Middle School
USA Drexel University
USA Duke University
USA Eastern Kentucky University
USA Eastmont High School
USA Fayetteville State University
USA Ferguson Enterprises
USA First Presbyterian Day School 
USA Florida International University
USA Francis Marion University
USA Gaston College
USA Georgia Institute of Technology
USA Glenpool Public School
USA Grand Valley State University
USA High School of Commerce
USA Hult International Business School
USA iFOSS
USA Indiana State University
USA Indiana University Bloomington
USA Ithaca College
USA Kingston university 
USA Lee University
USA Liberty High School
USA Liberty University
USA Louisiana State University
USA Luther College
USA Marin Juniors Volleyball Club
USA MGH Institute of Health Professions
USA MGH Institute of Health Professions
USA Michigan State University
USA Mount Mercy University
USA New Mexico State University
USA North Carolina Central University
USA North Carolina Central University 
USA North Carolina State University
USA Northeastern University
USA Northern Arizona University
USA Northern Illinois University
USA Ohio University
USA Old Dominion University
USA oneida-boces.org
USA Penn State University
USA Purdue University
USA Queensborough Community College
USA Regal Ent Group
USA Saginaw Valley State University
USA San Francisco State Unviersity
USA Security Forces Officer Course
USA Smiths Station Junior High School
USA Southern College of Optometry
USA Southern Illinois University Edwardsville
USA St. Joseph's Academy
USA St. Louis University
USA SUNY Buffalo
USA Taylor University
USA Tennessee Technological University 
USA Texas A&M Rangel College of Pharmacy
USA Texas A&M University
USA Texas A&M University-Corpus Christi
USA The College of Wooster
USA Thomas Jefferson Middle School
USA Tippie College of Business
USA Touro University
USA UC Berkeley
USA University of Alabama at Birmingham
USA University of Alaska-Fairbanks
USA University of Arizona
USA University of California, Irvine
USA University of Florida
USA University of Georgia
USA University of Illinois
USA University of Illinois at Chicago
USA University of Kansas
USA University of Kansas Medical Center
USA University of Maryland, Baltimore County
USA University of Massachusetts-Amherst
USA University of Memphis
USA University of Minnesota
USA University of Nebraska-Lincoln
USA University of North Carolina Greensboro
USA University of North Texas
USA University of Oregon
USA University of San Diego
USA University of South Alabama
USA University of Southern California
USA University of St. Thomas 
USA University of Texas at Arlington
USA University of Texas at Austin
USA University of Utah
USA University of Virginia
USA University of Wisconsin-Madison
USA Washington State University
USA Watts School of Nursing
USA Weber State University
USA West Shore Career Technical District
```

### PR DESCRIPTION
**Pull request by fairbet**
Fixes #6107 

Updated usermap

### CODE REVIEW

**Comment by junhaoyap** [comment]
Will look at this after https://github.com/TEAMMATES/teammates/pull/6100 is merged

**Comment by wkurniawan07** [comment]
@fairbet due to #6100, there are merge conflicts in this PR which you have to resolve.

**Comment by fairbet** [comment]
Conflicts are resolved!

**Comment by wkurniawan07** [comment]
@fairbet, please look at the [Files changed](https://github.com/TEAMMATES/teammates/pull/6122/files) introduced by this PR. There are so many things (in particular removal of institutions) that should not occur.

**Comment by junhaoyap** [comment]
@fairbet I believe you tried to introduce sorting or something similar, let's just add the missing ones and if we want to do sorting, we can create a new issue for that

**Comment by fairbet** [comment]
@junhaoyap @wkurniawan07 
Sorry for all these! I added all the missing ones, but there are still 56 deletions from re-sorting and adding missing commas. Please let me know if anything is still wrong!

**Comment by wkurniawan07** [comment]
@fairbet, following what @junhaoyap said, there should not be any form of re-sorting here. Can you undo those?

**Comment by fairbet** [comment]
@wkurniawan07  Ahh... I'm not sure how the list is originally sorted. Is it just alphabetical order? Because some entries are alphabetically sorted but some entries are not.

**Comment by fairbet** [comment]
@wkurniawan07  Fixed! Also fixed typos and deleted duplicate items!

File: src/main/webapp/js/userMapData.json [hunk 1]
```diff
@@ -468,13 +557,15 @@
...
     "Kingsmead College",
+    "Pearson Institute of Higher Education",
     "Stellenbosch University",
     "University of Pretoria",
     "University of the Western Cape"
   ],
   "South Korea": [
     "Harvard University",
     "Korea University Business School",
+    "Korea Unuversity",
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Typo?

Comment (fairbet) [inline src/main/webapp/js/userMapData.json]: Fixed!

File: src/main/webapp/js/userMapData.json [hunk 2]
```diff
@@ -199,47 +222,64 @@
     "ESC-Dijon",
     "Grenoble Ecole de Management",
     "HEC Paris",
+    "IESEG",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry after this, "IESEG School of Management".

File: src/main/webapp/js/userMapData.json [hunk 3]
```diff
@@ -199,47 +222,64 @@
...
+    "Francisco Marroquín University"
   ],
   "Hong Kong": [
     "CCC Rotary Secondary School",
     "HKU SPACE Community College",
     "Queen Elizabeth School",
     "The Hong Kong Polytechnic University",
-    "University of Hong Kong"
+    "University of Hong Kong",
+    "The ISF Academy",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This should be before "University of Hong Kong" (alphabetical order).

File: src/main/webapp/js/userMapData.json [hunk 4]
```diff
@@ -309,6 +359,7 @@
   "Malaysia": [
     "Kolej PolyTech Mara",
     "Sekolah Menengah Kebangsaan Bukit Jalor",
+    "Taylor's University",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry after this.

File: src/main/webapp/js/userMapData.json [hunk 5]
```diff
@@ -334,8 +386,10 @@
   ],
   "Netherlands": [
     "Amsterdam University College",
+    "Amsterdam University of Applied Science",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry after this.

File: src/main/webapp/js/userMapData.json [hunk 6]
```diff
@@ -348,31 +402,40 @@
...
     "Regius College Schagen",
     "ROC van Amsterdam",
+    "ROC van Twente",
     "Rotterdam School of Management",
     "Rotterdam University of Applied Science",
     "Saxion University",
     "Scala College Alphen aan den Rijn",
     "Summa College",
     "Teylingen College Leeuwenhorst",
+    "The Hague School of Applied Science",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry after this.

File: src/main/webapp/js/userMapData.json [hunk 7]
```diff
@@ -348,31 +402,40 @@
...
     "The School of Life",
     "TIAS Business School",
     "TU Delft",
+    "Universiteit Utrecht",
+    "University of Groningen",
     "University of Inholland",
     "University of Twente",
     "Utrecht University",
     "Vrije Universiteit Amsterdam",
+    "VU University",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry preceding this.

File: src/main/webapp/js/userMapData.json [hunk 8]
```diff
@@ -348,31 +402,40 @@
...
     "Saxion University",
     "Scala College Alphen aan den Rijn",
     "Summa College",
     "Teylingen College Leeuwenhorst",
+    "The Hague School of Applied Science",
     "The Hague University of Applied Sciences",
     "The School of Life",
     "TIAS Business School",
     "TU Delft",
+    "Universiteit Utrecht",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as "Utrecht University".

File: src/main/webapp/js/userMapData.json [hunk 9]
```diff
@@ -348,31 +402,40 @@
...
+    "University of Groningen",
     "University of Inholland",
     "University of Twente",
     "Utrecht University",
     "Vrije Universiteit Amsterdam",
+    "VU University",
     "Windesheim Honours College"
   ],
   "New Zealand": [
+    "Auckland University",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is not in the entry list?

File: src/main/webapp/js/userMapData.json [hunk 10]
```diff
@@ -395,12 +459,15 @@
     "Asian Institute of Management",
     "Ateneo de Manila University",
     "Ateneo School of Medicine & Public Health",
+    "Ateneo School of Medicine and Public Health",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry preceding this.

File: src/main/webapp/js/userMapData.json [hunk 11]
```diff
@@ -421,44 +488,66 @@
...
     "Sengkang Secondary School",
+    "Serangoon Junior College",
     "SHATEC Pte Ltd",
     "Silicon Straits",
     "Singapore Chinese Girls' School",
     "Singapore Institute of Technology",
     "Singapore Management University",
     "Singapore Polytechnic",
+    "St Gabriels' Secondary",
+    "ST MARGARET'S SECONDARY SCHOOL",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: Let's make this Pascal Case instead of UPPERCASE.

File: src/main/webapp/js/userMapData.json [hunk 12]
```diff
@@ -540,21 +634,25 @@
...
     "Cranfield University",
     "Edinburgh Napier School of Eng. and the Built Env.",
     "Edinburgh Napier University",
     "Emmanuel College",
     "Falmouth University",
+    "Henley Business School",
     "Home Learning College",
     "Hult International Business School",
     "Imperial College London",
+    "Kingston University",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry after this.

File: src/main/webapp/js/userMapData.json [hunk 13]
```diff
@@ -605,18 +706,22 @@
...
     "Bethany High School",
     "Black Hawk College",
+    "Bloomfield College",
     "Bluffton High School",
     "BMZA Portland Community College",
     "Boise State University",
     "Boston University",
     "Boston University School of Public Health",
     "Bowie Junior High",
-    "Bowling Green State University",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This should not be removed.

File: src/main/webapp/js/userMapData.json [hunk 14]
```diff
@@ -766,6 +892,7 @@
     "Kendall College School of Education",
     "Kennesaw State University",
     "Kent State University",
+    "Kingston university ",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: ~~u~~University

File: src/main/webapp/js/userMapData.json [hunk 15]
```diff
@@ -785,6 +913,9 @@
     "Louisiana State University",
     "Louis Pasteur Middle School 67",
     "Lower Columbia College",
+    "Luther College",
+    "Marin Juniors Volleyball Club",
+    "Mary Dutkiewicz",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is not in the entry list?

File: src/main/webapp/js/userMapData.json [hunk 16]
```diff
@@ -794,6 +925,7 @@
     "Mercer University",
     "Merrimack College",
     "MGH Institute for Health Professions",
+    "MGH Institute of Health Professions",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry preceding this, but since it looks like that this version (with "of") is the correct one, you can remove the one before.

File: src/main/webapp/js/userMapData.json [hunk 17]
```diff
@@ -862,6 +998,7 @@
     "Palmer High School",
     "Patrick County High School",
     "Paxton Consolidated Schools",
+    "Penn State University",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as "Pennsylvania State University".

File: src/main/webapp/js/userMapData.json [hunk 18]
```diff
@@ -926,25 +1072,36 @@
...
     "Sun Valley High School",
+    "SUNY Buffalo",
     "SUNY Jefferson Community College",
     "Syracuse University",
     "Tarleton State University",
     "Tarrant County College",
+    "Taylor University",
     "Temple University",
     "Tennessee Tech University",
+    "Tennessee Technological University",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the entry preceding this, but since the one spelled in full seems more appropriate, you can delete the entry before.

File: src/main/webapp/js/userMapData.json [hunk 19]
```diff
@@ -926,25 +1072,36 @@
...
+    "Texas A&M",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is not in the entry list?

File: src/main/webapp/js/userMapData.json [hunk 20]
```diff
@@ -954,11 +1111,13 @@
     "University of Alaska",
     "University of Alaska Anchorage",
     "University at Albany",
+    "University of Alaska-Fairbanks",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This should be before "University of Albany".

File: src/main/webapp/js/userMapData.json [hunk 21]
```diff
@@ -986,9 +1145,11 @@
     "University of Maryland",
     "University of Maryland Baltimore County",
     "University of Maryland University College",
+    "University of Maryland, Baltimore County",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the version without comma, two before this entry.

File: src/main/webapp/js/userMapData.json [hunk 22]
```diff
@@ -986,9 +1145,11 @@
...
     "University of Massachusetts",
     "University of Massachusetts Amherst",
     "University of Massachusetts Boston",
+    "University of Massachusetts-Amherst",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is the same as the version without hyphen, two before this entry.

File: src/main/webapp/js/userMapData.json [hunk 23]
```diff
@@ -1004,19 +1165,22 @@
...
     "University of Scranton",
+    "University of South Alabama",
     "University of South Carolina",
     "University of South Carolina Aiken",
     "University of South Dakota",
     "University of South Florida",
     "University of Southern California",
     "University of Southern Maine",
     "University of St. Thomas",
+    "University of St. Thomas ",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is duplicate of the entry before this.

File: src/main/webapp/js/userMapData.json [hunk 24]
```diff
@@ -1078,7 +1245,8 @@
     "Xavier University of Louisiana",
     "Yale University",
     "York College",
-    "Young harris College",
+    "Young Harris College",
+    "Your Group Getaways",
```
Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: This is not in the entry list?

**Comment by wkurniawan07** [comment]
@fairbet you also missed "IIS Bruno-Franchetti (Italy)".

**Comment by wkurniawan07** [comment]
@damithc I found that the following are not school/university/institute (not that we don't welcome them using TEAMMATES), should we include them in the list?
- Refugee Innovation Challenge (Germany)
- Independencia (Peru)
- Senior Interactive Academy (Romania)
- Fellowship of Christian Athletes (Singapore)
- Ministry of Education (Singapore)
- Cauldron Ice Cream, Inc. (USA)
- Ferguson Enterprises (USA)
- iFOSS (USA)
- Marin Juniors Volleyball Club (USA)
- oneida-boces.org (USA)
- Regal Ent Group (USA)
- Security Forces Officer Course (USA)

File: src/main/webapp/js/userMapData.json (hunk 9 above)
Comment (fairbet) [inline src/main/webapp/js/userMapData.json]: It was in June's list, but it's the same as University of Auckland!

File: src/main/webapp/js/userMapData.json (hunk 13 above)
Comment (fairbet) [inline src/main/webapp/js/userMapData.json]: There was a duplicate for this one.

File: src/main/webapp/js/userMapData.json (hunk 15 above)
Comment (fairbet) [inline src/main/webapp/js/userMapData.json]: Same thing, I also used June's list!

File: src/main/webapp/js/userMapData.json (hunk 19 above)
Comment (fairbet) [inline src/main/webapp/js/userMapData.json]: In June's list, but is a duplicate!

File: src/main/webapp/js/userMapData.json (hunk 24 above)
Comment (fairbet) [inline src/main/webapp/js/userMapData.json]: Was in June's list!

Comment (wkurniawan07) [inline src/main/webapp/js/userMapData.json]: @fairbet the issue title mentions "Jul-Sep 2016", which excludes June. You should not include whatever was included in the June's list in this PR. In general, when working on an issue, you should not include any unrelated fix unless they are tightly coupled.

**Comment by damithc** [comment]
@wkurniawan07 yes, we can include entities that are not universities/schools.

**Comment by fairbet** [comment]
@wkurniawan07 I fixed requested changes! Thank you for being so patient!

**Comment by wkurniawan07** [comment]
@fairbet looks good now, thanks for your contribution :+1:

If you wish to work on cleaning the institution list (e.g sorting), feel free to open an issue and work on it.
